.travis.yml

```diff
@@ -1,10 +1,18 @@
 language: julia
+os:
+  - linux
 julia:
-    - 0.5
-    - nightly
-script:
-    - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-    - julia -e 'Pkg.init(); Pkg.clone(pwd())'
-    - julia -e 'Pkg.test("NullableArrays", coverage=true)'
+  - 0.5
+  - 0.6
+  - nightly
+matrix:
+  allow_failures:
+    - julia: nightly
+notifications:
+  email: false
+#script:
```

Comment:
Commenting it allows the default script to run, which generally shouldn't be overridden unless there's a good reason to do so.